website/docs/getting-started/for-engineers.md

```diff
@@ -129,3 +129,14 @@ Import CSS helper classes by adding any of the following lines to the main Sass
 ```
 
 For more examples and guidelines read [the tokens documentation](/foundations/tokens).
+
+## Browser support
+
+The code of our styles, components and icons supports the following browsers:
```

Comment:
It almost feels like the causality is the other way around – browsers support features rather than the other way around – but I can't think of better wording.

Reply:
@alex-ju what do you think of this alternative?
```
Our styles, components and icons are supported by the following browsers:
```

Reply:
yep, that reads better in my opinion